### DISCUSSION TITLE
Can't see Write Contract option for verified contract on v2 frontend

### DISCUSSION THREAD

**Discussion by 0xDones** [blockscout/blockscout#8653] [Q&A] [answered]
When using Blockscout v1 frontend, I can see the option to write (call functions) to my contracts, but when using the v2 frontend I can't see it. Am I missing something?

### V1
![image](https://github.com/blockscout/blockscout/assets/29067941/0280aa9d-a923-40ef-9582-50567090fa2c)

### V2 (Separate Frontend)
![image](https://github.com/blockscout/blockscout/assets/29067941/52cd3d13-2922-4c25-acd5-7fc1390f88d0)

**Comment by ekawibisono** [comment, marked as answer]
Hi @refl3ction 

In the latest frontend, you must register your project in WalletConnect in order to use the Read Contract and Write Contract functions.

https://github.com/blockscout/frontend/blob/main/docs/ENVS.md#blockchain-interaction-writing-to-contract-etc

Register Your Project on Wallet Connect

`https://cloud.walletconnect.com/sign-up`
